.github/workflows/load_tests.yml

```diff
@@ -22,6 +22,15 @@ jobs:
           SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
           SLACK_USERNAME: AlertManager
           SLACK_ICON: https://a.slack-edge.com/80588/img/services/outgoing-webhook_48.png
-          SLACK_TITLE: K6 Load Tests has failed.
+          SLACK_TITLE: K6 load tests has failed.
           SLACK_MESSAGE: Some of the k6 load tests has failed.
-        if: failure()
\ No newline at end of file
+        if: failure()
+      - name: k6 Load Tests success notifications
```

Comment:
what if to use `SLACK_COLOR: ${{ job.status }}` instead of `failure()` ?
https://github.com/rtCamp/action-slack-notify

perhaps also: `SLACK_MESSAGE: k6 load tests result is ${{ job.status }}`
https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#job-context
```
job.status | string | The current status of the job. Possible values are success, failure, or cancelled.
```


`